spec/services/slack/messengers/feedback_spec.rb

```diff
@@ -0,0 +1,84 @@
+require "rails_helper"
+
+RSpec.describe Slack::Messengers::Feedback, type: :service do
+  let_it_be_readonly(:user) { create(:user) }
+  let(:default_params) do
+    {
+      type: "abuse-reports",
+      category: "spam",
+      reported_url: "http://example.com",
+      message: "this is spam"
+    }
+  end
+
+  def get_argument_from_last_job(argument_name)
+    job = sidekiq_enqueued_jobs(worker: SlackBotPingWorker).last
+    job["args"].first[argument_name]
+  end
+
+  it "supports an anonymous report" do
+    sidekiq_assert_enqueued_jobs(1, only: SlackBotPingWorker) do
+      described_class.call(default_params)
+    end
+  end
+
+  it "contains user's details", :aggregate_failures do
+    sidekiq_assert_enqueued_jobs(1, only: SlackBotPingWorker) do
+      described_class.call(default_params.merge(user: user))
+    end
+
+    message = get_argument_from_last_job("message")
+
+    url = "#{ApplicationConfig['APP_PROTOCOL']}#{ApplicationConfig['APP_DOMAIN']}/#{user.username}"
```

Comment:
I think the extra line on `31` should come after the `url` variable declaration, no?

Reply:
done